crates/protocol/src/lib.rs

```diff
@@ -36,11 +36,7 @@ pub trait Validator:
 
 /// Types for converting to and from protobuf
 pub mod protobuf {
-    #![allow(clippy::all)]
-    // Generated by [`prost-build`]
-    include!(concat!(env!("OUT_DIR"), "/warg.protocol.rs"));
-    // Generated by [`pbjson-build`]
-    include!(concat!(env!("OUT_DIR"), "/warg.protocol.serde.rs"));
+    pub use warg_protobuf::protocol::*;
 
     pub fn prost_to_pbjson_timestamp(timestamp: prost_types::Timestamp) -> pbjson_types::Timestamp {
```

Comment:
These helpers should presumably migrate to the protobuf crate as well, but that can be done separately.